/**: # namespace and node name
  ros__parameters:
    qos_depth: 30
    num_target: 3
    pick_target: 325
    max_speed : 2.0  # [m/s]
    min_speed : -0.5  # [m/s]
    max_yaw_rate : 0.69777 # 40.0 * math.pi / 180.0  # [rad/s]
    max_accel : 0.2  # [m/ss]
    max_delta_yaw_rate : 0.8 # [rad/ss]
    v_resolution : 0.01  # [m/s]
    yaw_rate_resolution : 0.0017444 # 0.1 * math.pi / 180.0  # [rad/s]
    dt : 0.2  # [s] Time tick for motion prediction
    predict_time : 4.0  # [s]
    to_goal_cost_gain : 2.0
    speed_cost_gain : 1.0
    obstacle_cost_gain : 0.5
    robot_stuck_flag_cons : 0.1  # constant to prevent robot stucked
    robot_type : 0 # 0 for circle, 1 for rectangle